### PR TITLE
Intergrating clib-uninstall

### PR DESCRIPTION
**Pull request by Isty001**
Moving clibs/clib-uninstall into clib.

### CODE REVIEW

**Review by jwerle — APPROVED** [review]

**Comment by Isty001** [comment]
For the future, clibs/http-get.c #7 was related to the failing test on Windows

File: src/clib-uninstall.c [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  char *pkg = NULL;
+  const char *val = NULL;
+  JSON_Value *root = NULL;
+  JSON_Object *obj = NULL;
+
+  size = asprintf(&dir, "/tmp/%s-%s", name, version);
+  if (-1 == size)
+    return NULL;
+
+  size = asprintf(&pkg, "%s/package.json", dir);
```
Comment (stephenmathieson) [inline src/clib-uninstall.c:86]: I don't think we should merge this as-is. All of the existing commands support `clib.json`. For good UX, we should keep things consistent between commands.

IMO we have two options:
1. (easy) just replace `package.json` with `clib.json` and mention that `clib uninstall` doesn't support the old manifest name. This isn't the preferred solution, but could be done quickly, enabling us to move on to other features.
2. Update this to _first_ check for `clib.json` and fallback to the `package.json` manifest. This is what the other commands do, but is a bit more work.

WDYT?

**Review by stephenmathieson — CHANGES_REQUESTED** [review]

File: src/clib-uninstall.c (hunk 1 above)
Comment (Isty001) [reply src/clib-uninstall.c:86]: Yes, you are right! I completely overlooked this. I think the 2. option is more correct, as clib has this behavior elsewhere.

**Review by stephenmathieson — APPROVED** [review]
One small thing, but not worth holding this up. LGTM!

**Pull request merged**